lib/alchemy/resource.rb

```diff
@@ -220,16 +220,17 @@ def skipped_attributes
 
     private
 
-    def string_attribute?(a)
-      a[:type].to_sym == :string && !a.key?(:relation)
+    def string_or_text_attribute?(a)
+      (a[:type].to_sym == :string || a[:type].to_sym == :text) && !a.key?(:relation)
```

Comment:
Here I would introduce an new constant `SEARCHABLE_COLUMN_TYPES` and ask if the attribute type is included in this array. This would reduce the length of this method body and will introduce a expandable interface for future additions.